cmd/lakectl/cmd/dbt_util.go

```diff
@@ -41,14 +26,16 @@ type CommandExecutor interface {
 	ExecuteCommand(*exec.Cmd) ([]byte, error)
 }
 
+var errSchemaExtraction = fmt.Errorf("failed extracting schema from dbt debug message")
+
 func ValidateGenerateSchemaMacro(projectRoot, macrosDirName, generateSchemaFileName, schemaIdentifier string) error {
 	p := path.Join(projectRoot, macrosDirName, generateSchemaFileName)
 	data, err := ioutil.ReadFile(p)
 	if err != nil {
 		return err
 	}
 	if !strings.Contains(string(data), schemaIdentifier) {
-		return MissingSchemaIdentifierError{schemaIdentifier: schemaIdentifier, generateSchemaFileName: generateSchemaFileName}
+		return fmt.Errorf("generate_schema_name does not contain %s addition. Handle lakefs support to %s or use skip-views flag: %w", schemaIdentifier, generateSchemaFileName, err)
```

Comment:
I don't understand this error message, specifically "Handle lakefs support".

In general error messages should not contain more than a single phrase, because `Errorf` will be used on them.